README.md

```diff
@@ -47,6 +47,11 @@ $ vim .devcontainer/Dockerfile
 
 Exit the container, and re-run the `docker run` command... after the build completes, `htop` should exist in the container! 🥳
 
+> **Note:** Envbuilder performs destructive filesystem operations! To guard against accidental data
+> loss, it will refuse to run if it detects that KANIKO_DIR is not set to a specific value.
+> If you need to bypass this behaviour for any reason, you can bypass this safety check by setting
+> `FORCE_SAFE=true`.
```

Comment:
I'd love for us to start using the `ENVBUILDER_` prefix for our envs, can we start now?

Reply:
I'll do this in a separate PR. This isn't a new option, it's been there all the time but not doing anything.